services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -514,9 +514,10 @@ async def delete_project_simcore_s3(
                 await db_file_meta_data.delete_all_from_project(conn, project_id)
             else:
                 await db_file_meta_data.delete_all_from_node(conn, node_id)
-            await get_s3_client(self.app).delete_files_in_project_node(
-                self.simcore_bucket_name, project_id, node_id
-            )
+
+        await get_s3_client(self.app).delete_files_in_project_node(
+            self.simcore_bucket_name, project_id, node_id
+        )
```

Comment:
hmm, ok. now comes the question.
What if the S3 client raises for whatever reason?? then the database lose synchronisation right?

Reply:
yes this will not bring anything. reverted and added more logging for now.